{"uid":"e698368f2fe5bb35","name":"An uncaught error was detected outside of a test","fullName":"An uncaught error was detected outside of a test","historyId":"c59643030e1e776644fd5d1d6aa79cfc","time":{"start":1681280958820,"stop":1681280962413,"duration":3593},"status":"failed","statusMessage":"The following error originated from your test code, not from Cypress.\n\n  > undefined\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","statusTrace":"Error: The following error originated from your test code, not from Cypress.\n\n  > undefined\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"package","value":"cypress\\e2e\\book-a-seat-api.cy.js"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"e698368f2fe5bb35.json","parameterValues":[]}